examples/web-server/nginx/README.md

```diff
@@ -0,0 +1,104 @@
+# How to use NGINX as a reverse-proxy with LetsEncrypt
+
+## Requirements
+
+1. You'll need a subdomain and the a wildcard subdomain configured that resolves to server.
```

Comment:
I noticed you used `1, 2, 3` for requirements but bullet points for all of the other steps. Can you use numbers everywhere? We used numbers in the Caddy one, so there's a good pattern for that.